src/GoTrueClient.ts

```diff
@@ -2355,7 +2357,12 @@ export default class GoTrueClient {
           return { data: null, error }
         }
 
-        if (data?.totp?.qr_code) {
+        // TODO: Remove once: https://github.com/supabase/auth/pull/1717 is deployed
```

Comment:
Made a mistake here - it still shows up in the response field. With the above change it should no longer show up in the response field for phone responses - till then we can remove the field for phone

It's unlikely that anyone will use (and depend) on the feature via the API given how new the feature is.